### ISSUE TITLE
Feature Request: Event Scratchpad

### ISSUE THREAD

**Issue by rabimba** [techspeakers/hub#3]
In response to #1 

Proposing a new feature as part of the Hub dashboard.

### Event Scratchpad

- Input_ should be linked to tweets/toots about events or more specifically links to images taken during an event/talk relevant to the Tech Speaker activity. _Input_ would only be visible in the speaker/event organizer view, from the authenticated user who submitted the links
- ScratchPad_ Any tweet/toot/image link posted via _inpput_ view will be expanded to show the relevant images of the event. This would be visible to everybody.  tweet/toot/image should follow specific convention with event HashTag and scratchpad can take that to show the images eventwise. Everyone apart from poster should be able to see this to boost/or just for later reporting purposes.

_For now I propose this to be expanded only for the day those images/tweets are posted. And hidden to an archive on future dates and only used later for reporting purposes._

**Comment by flaki** [comment]
I was thinking on implementing a very similar feature, I call it "notes" but "scratchpad" is pretty close to what it is, too. It has multiple usecases, one of them being used for reporting later.

Notes is a button that is accessible from the main screen of the app, and at the very basic implementation comes with a text area and a button.

The number one concern is usability: making it possible for TechSpeakers to quickly jot down everything. Links, feedback, requests, everything that might be interesting or in need to be followed up on. Notes are saved locally (so slow speed, or lack of wifi connection won't cause issues), and are synced to the server once connection resumes. Since these notes are saved with timestamps (and the app knows about the events of the user from the activity calendar) later on these notes can be recalled for the submission of the event report (and for any particular followups).

Later on, based on usage, we can add other affordances, but even like this it would be very helpful for lot of us.

How do you see your Scratchpad idea relate to the above @rabimba?

**Comment by rabimba** [comment]
I can see both of them nicely being integrated. 
The notes will act as....well notes. So anything in that freeform text field will appear as....text and saved.
Anything with a hyperlink (and our predefined domains like twitter/mastodon) will come up as a tiled card (tweet embed) for others to boots/rt.
So UI input implementation wise a freeform text field looks nice to me.

**Comment by flaki** [comment]
Yes it's definitely possible to use these snippets in the UI and expose them in other places. It would also be trivial to expose a button for the note author to instantly request boosting from official accounts (e.g. [TS Twitter](https://on.tchspk.rs/tweets)), or propose it to be featured in the newsletter (if it's e.g. a speaking announcement or a recorded video).
